### ISSUE TITLE
Wrong error is highlighted

### ISSUE THREAD

**Issue by erbridge** [SublimeLinter/SublimeLinter-jshint#74]
```js
result.get('me').then(function(me) {});
```

The first `me` is highlighted, not the second.

`jshint` gives: `file.js: line XX, col YY, 'me' is defined but never used.` where YY is the correct column for the second `me`. This is also what appears in the status bar in Sublime.

**Comment by fiznool** [comment]
Seeing the same issue:

![screenshot from 2016-01-27 15-32-04](https://cloud.githubusercontent.com/assets/820863/12618296/29a546c4-c50b-11e5-9102-2ba6e2f0d624.png)

I think this is related to https://github.com/SublimeLinter/SublimeLinter3/issues/188

**Comment by duzun** [comment]
Encountered the same issue:

```js
var xhr = $.ajax({...}).done(function (data, status, xhr) { ... });
```

First `xhr` highlighted, where in fact the second one should be highlighted.

**Comment by braver** [comment]
I can't readily reproduce this, does it still happen with recent versions?

**Comment by braver** [comment]
@duzun Can you give me an example file and jshintrc (if any)?

**Comment by kaste** [comment]
I have a PR #116

**Comment by braver** [comment]
@duzun Thanks!

@kaste I can confirm #116 fixes the issue.

**Comment by kaste** [comment]
🤹‍♀️  And it only took 3 years! 🤹‍♂️
